docs/i18n/zh/docusaurus-plugin-content-docs/current/reference/sdk/evaluation.md

```diff
@@ -184,12 +209,33 @@ def func():
 |`task_num`|step会被分成task的数量|
 |`needs`|依赖的step列表|
 
-`resources` 格式为 {名称}:{数量}。名称为资源的种类，目前支持 `cpu`、`gpu` 和 `memory`。当种类为 `cpu` 时，数量的类型为float, 没有单位，1表示1个cpu core，对应Kubernetes resource的request；当种类为 `gpu` 时，数量的类型为int，没有单位，1表示1个gpu，对应Kubernetes resource的request和limit；当种类为 `memory`时，数量的类型为float，没有单位，1表示1MB内存，对应Kubernetes resource的request。`resources` 使用列表的方式支持指定多个资源，且这些资源都满足时才会进行调度。当不写 `resources` 时，会使用所在Kubernetes的cpu、memory默认值。 `resources` 表示的是一个task执行的时所需要的资源情况，并不是step所有task的资源总和限制。**目前 `resources` 只在Cloud Instance中生效**。 `resources` 使用例子如下：
+`resources` 格式为：
+- 简化表达方式
+  ```json
+  {
+    {名称}:{数量},
+    ...
+  }
+  ```
+  代表request和limit同时设置，且值相同。
+- 完全表达方式
+  ```json
```

Comment:
why use JSON format for users?

Reply:
it's a doc format, for rendering as a dict

Reply:
done